kerastuner/engine/base_tuner.py

```diff
@@ -220,23 +220,46 @@ def on_search_end(self):
             self.logger.exit()
 
     def get_best_models(self, num_models=1):
-        """Returns the best model(s), as determined by the tuner's objective.
+        """Returns the best model(s), as determined by the objective.
 
         This method is only a convenience shortcut. For best performance, It is
         recommended to retrain your Model on the full dataset using the best
         hyperparameters found during `search`.
 
-        Args:
-            num_models (int, optional): Number of best models to return.
+        # Arguments:
+            num_models (int, optional). Number of best models to return.
                 Models will be returned in sorted order. Defaults to 1.
 
-        Returns:
+        # Returns:
             List of trained model instances.
         """
         best_trials = self.oracle.get_best_trials(num_models)
         models = [self.load_model(trial) for trial in best_trials]
         return models
 
+    def get_best_hyperparameters(self, num_trials=1):
+        """Returns the best hyperparameters, as determined by the objective.
+
+        This method can be used to reinstantiate the (untrained) best model
+        found during the search process.
+
+        Example:
+
+        ```python
+        best_hp = tuner.get_best_hyperparameters()[0]
+        model = tuner.hypermodel.build(best_hp)
+        ```
+
+        # Arguments:
+            num_trials: (int, optional). Number of `HyperParameters` objects to
+              return. `HyperParameters` will be returned in sorted order based on
+              trial performance.
+
+        # Returns:
+            List of `HyperParameter` objects.
+        """
+        return [t.hyperparameters for t in self.oracle.get_best_trials(num_trials)]
```

Comment:
Why not just return the `values` dict? This is what people are going to be looking for most of the time. If looking for the actual hp object, they can get it easily from `get_best_trials`.

Reply:
The `values` dict by itself doesn't have logic to handle conditional parameters, so if those were present it couldn't be used to recreate the best model. My thinking was that the main usecase for this method would be to pass its results to `build_model` to retrain the best model on the full dataset, and `build_model` expects the `HyperParameters` object

Reply:
Ok, makes sense.